src/components/basic/SearchInput/SearchInput.mdx

```diff
@@ -0,0 +1,81 @@
+{/**\*\*\*\***\*\***\*\*\*\***\*\*\*\***\*\*\*\***\*\***\*\*\*\***\***\*\*\*\***\*\***\*\*\*\***\*\*\*\***\*\*\*\***\*\***\*\*\*\***
```

Comment:
this is a kinda strange way to set comments are you sure about that? this comment applies to all mdx files

Reply:
this is auto generated after running ```yarn pretty```